build.gradle

```diff
@@ -103,6 +103,7 @@ def createSpotlessTarget = { pattern ->
         'secrets',
         'charts', // Helm charts often have injected template strings that will fail general linting. Helm linting is done separately.
         'resources/seed/*_specs.yaml', // Do not remove - this is necessary to prevent diffs in our github workflows, as the file diff check runs between the Format step and the Build step, the latter of which generates the file.
+        'airbyte-integrations/connectors/source-amplitude/unit_tests/api_data/zipped.json', // Zipped file presents as non-UTF-8 making spotless sad
```

Comment:
dear reviewer this is the only line that isn't formatting only

Reply:
looks great
I wonder why we have such a file though lol

Reply:
yeah I didn't blame it, I'll ask whomever is the author